### DISCUSSION TITLE
好書分享<數位時代的人權思辨>

### DISCUSSION THREAD

**Discussion by Tomlord1122** [1101-GS4719-C/Python-Community-GS4719-C#194] [General]
本書在探討資訊科技影響我們的有甚麼，像是以youtube、facebook為例，我們在搜尋欄查了什麼東西，我們看了甚麼影片，那這些軟體內部的演算法，就會推薦相同主題或是類似的文章影片給我們看，長期下來，我們人就會有一種被同樣概念給重複洗腦的問題，甚至有很多政黨概念會加諸其中，假新聞的出現，也會影響我們的價值判斷，所以要如何有一個好的媒體識讀能力是我們這一代人需要面對的重大課題，我認為要有獨立思考的能力，資訊量龐大，我們要能過濾篩選掉不好的內容，且不要有非黑即白的看法。

https://www.books.com.tw/products/0010865368
購書連結大家有興趣可以看看

**Comment by HSU-YU-JEN** [comment]
在這種演算法的操作下，我們也很容易待在自己的同溫層裡，如此下去，就會連帶著視野的限縮與缺乏與不同立場的人思辨與溝通

**Reply by Jie2222** [reply]
覺得滿有趣也滿諷刺的，在這個數位時代，我們感覺我們好像擁有無限多的資訊，但卻在不知不覺中被限縮在演算法操控的狹小世界裡。

**Comment by vvvioletta** [comment]
這次的全國性公投也可以讓我們反思，決定同意與不同意的背後，我們真的做足功課，而吸收的資訊裡頭包含夠多各種聲音觀點了嗎？
或者真的只是被某部分的政治意識型態影響、有話語權的媒體刻意操縱，然後就投下四個同意、四個不同意。如果對於單一議題不了解，除了同意與不同意，甚至有不投的選項。
回歸同學資訊科技和媒體識讀的部分，之前英國脫歐公投就有臉書透過演算法公布偏差的民調新聞進而影響中間選民出來投脫歐的新聞，我想這也體現了當今資訊科技影響媒體的一個例子。

**Comment by ChinChun0220** [comment]
演算法的幫助下，我們更容易待在自己的同溫層，甚至會無意間把同溫層加厚（取消關注不同意見的人們），而當脫離同溫層時會帶來更大的衝擊，沒玩過幾個論壇，但發現每個地方都有不同的同溫層覺得好可怕🥲
最主要是要學習包容與傾聽的能力去瞭解與接受不同想法的人

**Comment by 107602505** [comment]
演算法可以用於解決某一個特定的問題，為生活帶來許多的便利且也證明科技正快速的改變著世界，在短時間內可能會因為吸收類似的資訊而被影響，所以要多方接收不同的想法、知識，而且最重要的是要會獨立思考才不會限縮在演算的狹小世界中。

**Comment by 109302003** [comment]
我最近都利用演算法逛網拍，想買特定的商品例如短靴，只要去搜尋引擎搜個兩次，fb 和ig就會推給我相關網拍的廣告，如果我又點進去商家廣告，之後就都會是同一類型網拍或商品，方便我比較價錢及品質。

**Comment by syt-110** [comment]
演算法容易讓人活在同溫層裡(例如fb)，雖然比較方便，但是很難知道大部分或小眾等不同於自己的意見，假消息也猖獗。
